src/odemis/acq/align/test/clspots_optical_autofocus_test.py

```diff
@@ -44,34 +44,14 @@ class TestAutofocusSim(unittest.TestCase):
     """
     Test simulated auto focus for CL spots.
     """
-    backend_was_running = False
 
     @classmethod
     def setUpClass(cls):
-        try:
-            test.start_backend(CLSPOTS_SIM_CONFIG)
-        except LookupError:
-            logging.debug("A running backend is already found, skipping tests")
-            cls.backend_was_running = True
-            return
-        except IOError as exp:
-            logging.error(str(exp))
-            raise
+        test.start_backend(CLSPOTS_SIM_CONFIG)
```

Comment:
you don't want to run these tests if you're connected to the hardware, so shouldn't there be a 
```
if not TEST_NOHW:
    skip test
```

Reply:
Why do we not want to run them with the HW?